index.rst

```diff
@@ -15,5 +10,241 @@ Change Record
+
+* Level 1 denotes the most important milestones exposed at the NSF level
+
+* Level 2 denotes cross-subsystem milestones (for example, DM milestones that affect the Camera Subsystem)
+
+* Level 3 denotes cross-team milestones within a subsystem (for example, Middleware milestones that effect the DRP Team)
+
+* Level 4 denotes internal milestones within a team.
+
+*O(few hundred)* milestones have been defined to-date to track the progress of DM activities.
+
+Relationship between milestones, as well as between milestones and planning packages are captured:
+milestones typically *block* planning packages, milestones can *block* other milestones, or they
+can *relate* to other milestones.
+
+Baseline Long-Term Plan
+-----------------------
+
+Planning packages, together with milestones form the *Baseline Plan*. The plan is under change control.
+
+The plan is stable. Any change to the plan, including changing scope, cost or schedule, must be approved
+by the appropriate change control body:
+
+* Change Control Board approval for any non-trivial change to a planning package or Level 1 and
+  Level 2 milestone
+
+* DM's TCT approval for Level 3 milestones
+
+* no approval is needed for Level 4 milestones, however any non-trivial change must be communicated
+  to the DM team
+
+* trivial changes, such as small corrections / clarifications to narrative that do not affect
+  scope, time or budget are allowed without approval.
+
+Typically, adjustments to Level 1, 2 and 3 milestones are made every 6 or 12 months. Changes to Level 4
+milestones can occur more frequently.
+
+Short-Term Planning
+-------------------
+
+The short-term plan is managed through *epics* and *stories*, and executed through *cycles*.
+
+In the EVMS world, epics map directly to *activities*, and stories map to *activity steps*.
+
+Epics and Stories
+~~~~~~~~~~~~~~~~~
+
+Typically, planning packages encompass relatively large blocks of work. To plan and execute work
+in details, finer-grain planning is required. This is implemented through epics and stories.
+
+Each epic captures a non-trivial work that is associated with a subset of work defined in a
+planning package. Each epic has a clear description, well defined deliverables, and
+relationships with a planning package (an epic *implements* a PP), and a milestone (an epic
+*blocks* a milestone). Epics are resource loaded, and have start and end dates defined.
+
+Each epic is broken into smaller chunks of work, called *stories*. Stories are primarily used
+to define and manage the short-term activities of individual developers.
+
+Epics and stories are used to track all software work, as well as work on delivering hardware and
+services that are tightly coupled with software (example: "Deliver OpenStack-based Test Cluster
+running RedHat 6.1".) Details of work on standalone activities do not have to be captured on daily
+bases through stories (example: "Deliver 10Gbit link between Chile and USA")
```

Comment:
Fixed the typo.

Extrapolating from the other text, I've changed this to read "Work which has no direct relationship to software development do not have to be flowed down to stories". I think that's a marginal improvement, but I'm still not sure I understand the thinking here. @jbecla, can you clarify?